server/data/data-jmap/src/test/java/org/apache/james/jmap/api/upload/UploadRepositoryContract.scala

```diff
@@ -217,18 +218,4 @@
      assertThat(SMono.fromPublisher(testee.retrieve(uploadId2, USER)).block())
        .isNotNull
    }
-
-   @Test
-   def deleteShouldReturnTrueWhenRowExists(): Unit = {
```

Comment:
no, please keep it

Reply:
https://github.com/apache/james-project/blob/master/server/data/data-jmap/src/test/java/org/apache/james/jmap/api/upload/UploadRepositoryContract.scala#L192

Reply:
It is already there, messed up during the rebase, it was duplicated, thus not compiling. That's just removing the duplicate tests